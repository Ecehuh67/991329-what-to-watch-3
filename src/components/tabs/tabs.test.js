import Tabs from './tabs';

const state = {
  activeTab: `Due`,
  tabsList: [`Due`, `Vue`],
  listener: () => {}
};

const emptyData = {
  activeTab: ``,
  tabsList: [``, ``],
  listener: () => {}
};

describe(`Render Tabs correctly`, () => {
  it(`Render empty data`, () => {
    const tree = renderer.create(
        <Tabs
          state={emptyData}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
  it(`Render mock data the right way`, () => {
    const tree = renderer.create(
        <Tabs
          state={state}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
