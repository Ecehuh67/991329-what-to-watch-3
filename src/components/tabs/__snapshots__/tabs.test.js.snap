// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Tabs correctly Render empty data 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item movie-nav__item--active"
      onClick={[Function]}
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        
      </a>
    </li>
    <li
      className="movie-nav__item movie-nav__item--active"
      onClick={[Function]}
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        
      </a>
    </li>
  </ul>
</nav>
`;

exports[`Render Tabs correctly Render mock data the right way 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item movie-nav__item--active"
      onClick={[Function]}
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        Due
      </a>
    </li>
    <li
      className="movie-nav__item"
      onClick={[Function]}
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        Vue
      </a>
    </li>
  </ul>
</nav>
`;
